src/reader.rs

```diff
@@ -2787,6 +2787,24 @@ impl<'a> Reader<'a> {
                     self.compute_and_apply_completions(c);
                 }
             }
+            rl::PagerShowSearch => {
```

Comment:
yeah some of our special readline functions are inflexible DWIM-style commands.
For this case, does the approach from https://github.com/fish-shell/fish-shell/issues/11193#issuecomment-2692821049
work for you? I guess that one is a bit backwards and unintuitive.

Reply:
Yes, it works!

And since there's already a workaround, I think an alternative to this PR could be adding it to the documentation?